ess/src/API/EMBC.ESS/Managers/Submissions/SubmissionsManager.cs

```diff
@@ -170,6 +176,11 @@ public async Task<string> Handle(SaveRegistrantCommand cmd)
             var contact = mapper.Map<Contact>(cmd.Profile);
             var result = await contactRepository.ManageContact(new SaveContact { Contact = contact });
 
+            if (cmd.Profile.SecurityQuestions.Where(s => s.AnswerChanged).Any())
```

Comment:
what would happen if one is changed, and other are not? should there be a select for the changed questions and then call to update only the modified ones?